src/controllers/dlgprefcontroller.cpp

```diff
@@ -543,6 +546,25 @@ void DlgPrefController::slotPresetLoaded(ControllerPresetPointer preset) {
     }
 }
 
+void DlgPrefController::setupWarningLabel(ControllerPresetPointer preset) {
+    if (presetIsSupported(preset)) {
+        m_ui.groupBoxWarning->hide();
+    } else {
+        m_ui.groupBoxWarning->show();
+        m_ui.labelWarning->setText(tr("<font color='#BB0000'><b>If you use this preset your controller may not work correctly. "
+                "Please select another preset.</b></font><br><br>"
```

Comment:
"Please select another preset or disable the controller."

Reply:
Good! Done.